docs/source/reference/storage.rst

```diff
@@ -245,10 +260,10 @@ Storage YAML reference
 
       sky.Storage.source: str
         The source attribute specifies the local path that must be made available
-        in the storage object. It can either be a local path, in which case data
-        is uploaded to the cloud to an appropriate object store (s3 or gcs), or it
-        can be a remote path (s3://, gs://), in which case it is copied or mounted
-        directly (see mode flag below).
+        in the storage object. It can either be a local path or a list of local
+        paths, in which case data is uploaded to the cloud to an appropriate
```

Comment:
Grammar: `or a list of local paths. Data is uploaded to the cloud to ....`

Reply:
Fixed!